### PR TITLE
Swahili Translation

### PR DESCRIPTION
**Pull request by Pheogrammer**
Translation of English to Swahili for Laravel Language

### CODE REVIEW

File: locales/sw/sw.json [hunk 1]
```diff
@@ -84,160 +83,160 @@
...
     "Central African Republic": "Jamhuri Ya Afrika Ya Kati",
     "Chad": "Chad",
-    "Change Subscription Plan": "Change Subscription Plan",
+    "Change Subscription Plan": "Badilisha Mpango wa Usajili",
     "Changes": "Mabadiliko",
     "Chile": "Chile",
     "China": "China",
     "Choose": "Kuchagua",
-    "Choose :field": "Kuchagua :field",
+    "Choose :field": "chagua :field",
```
Comment (andrey-helldar) [inline locales/sw/sw.json:124]:
```suggestion
    "Choose :field": "Chagua :field",
```

File: locales/sw/sw.json [hunk 2]
```diff
@@ -84,160 +83,160 @@
...
     "Christmas Island": "Kisiwa Cha Krismasi",
-    "City": "City",
-    "click here to request another": "click here to request another",
+    "City": "Jiji",
+    "click here to request another": "bonyeza hapa kuomba nyingine",
     "Click to choose": "Bonyeza kuchagua",
-    "Close": "Karibu",
-    "Cocos (Keeling) Islands": "Cocos (Keeling) Islands",
+    "Close": "Funga",
+    "Cocos (Keeling) Islands": "Visiwa vya Cocos (Keeling).",
```
Comment (andrey-helldar) [inline locales/sw/sw.json:135]:
```suggestion
    "Cocos (Keeling) Islands": "Visiwa vya Cocos (Keeling)",
```

File: locales/sw/sw.json [hunk 3]
```diff
@@ -306,69 +305,69 @@
...
-    "Moldova, Republic of": "Moldova, Republic of",
+    "Moldova, Republic of": "Moldova, Jamhuri ya",
     "Monaco": "Monaco",
     "Mongolia": "Mongolia",
     "Montenegro": "Montenegro",
     "Month To Date": "Mwezi Na Tarehe",
-    "Monthly": "Monthly",
-    "monthly": "monthly",
+    "Monthly": "Kila mwezi",
+    "monthly": "Kila mwezi",
```
Comment (andrey-helldar) [inline locales/sw/sw.json:370]:
```suggestion
    "monthly": "kila mwezi",
```

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Comment by andrey-helldar** [comment]
@Pheogrammer, do you want to join our team?

Each of us is engaged in the fact that he monitors the chosen localization. When you need to fix something or new keys have been added - translate them. When someone creates a PR for this localization, check it out.

Usually when someone submits a localization PR, @caouecs or I assign it to the users to check against the [referent](https://laravel-lang.com/referents.html) table.

Interesting?

**Comment by Pheogrammer** [comment]
> @Pheogrammer, do you want to join our team?
> 
> Each of us is engaged in the fact that he monitors the chosen localization. When you need to fix something or new keys have been added - translate them. When someone creates a PR for this localization, check it out.
> 
> Usually when someone submits a localization PR, @caouecs or I assign it to the users to check against the [referent](https://laravel-lang.com/referents.html) table.
> 
> Interesting?

Sure, I would love to 💯

**Comment by andrey-helldar** [comment]
@caouecs, we have a new team member. Can you add him to the team?

**Comment by caouecs** [comment]
done

Welcome @Pheogrammer

**Pull request merged**